### DISCUSSION TITLE
Restore Data without DB

### DISCUSSION THREAD

**Discussion by hoamer** [gogs/gogs#7496] [Q&A] [answered]
Hello everyone

I was using gogs a long time ago. Unfortunately my database for gogs isnt there anymore.
But I have found data at "gogs-repositories":
```
drwxr-xr-x  2 geoclue avahi 4096 17. Okt 2018  branches
-rw-r--r--  1 geoclue avahi   66 17. Okt 2018  config
-rw-r--r--  1 geoclue avahi   73 17. Okt 2018  description
-rw-r--r--  1 geoclue avahi   23 17. Okt 2018  HEAD
drwxr-xr-x  2 geoclue avahi 4096 17. Okt 2018  hooks
drwxr-xr-x  2 geoclue avahi 4096 28. Feb 2019  info
drwxr-xr-x 78 geoclue avahi 4096 28. Feb 2019  objects
drwxr-xr-x  4 geoclue avahi 4096 17. Okt 2018  refs
```

**Question**
(How) is it possible to get the repository as a "normal" git repository?
I would also appreciate any recommendation how to get the code/repo back.

Best regards
Sebastian

**Comment by annpocoyo** [comment, marked as answer]
You can run it as a local repository. You should just be able to specify the git repository folder as a path to git clone like this:`git clone path/to/gogs-repository/user/repo.git/` . Then fire up a new gogs instance and create and push the just cloned git repository to it.

**Reply by hoamer** [reply]
It worked like a charm. Thank you!
